A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2420,6 +2420,11 @@ class FMCMainDisplay extends BaseAirliners {
         }
     }
 
+    setPerfApprFlaps3(s) {
+        this.perfApprFlaps3 = s;
+        return true;
+    }
+
```

Comment:
```suggestion
    setPerfApprFlaps3(s) {
        this.perfApprFlaps3 = s;
        SimVar.SetSimVarValue("L:A32NX_SPEEDS_LANDING_CONF3", "boolean", s);
    }
```

Reply:
no need for a return

Reply:
Ack on the ret... should have removed it once I didn't need it.

Good catch on the simvar. I hadn't seen anything in the MCDU or FCOM to suggest there would be anything else needing to know about it so I didn't go looking... but I missed the PFD F speed marker: https://github.com/flybywiresim/a32nx/blob/5160ea16c15868251f97ba4489304f328caafa8c/A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js#L1154. I still can't find a reference for the F speed marker disappearing when LDG CONF 3 is selected...